CHANGELOG.md

```diff
@@ -6,9 +6,11 @@ Not yet released; provisionally v1.3.3 (may change).
 
 ### Map Changes
 
-The verifiable map is still experimental. APIs have been deprecated and will be
-deleted in the near future. These changes will not affect the Trillian module
-semantic version due to the experimental status of the Map.
+The verifiable map is still experimental. APIs, such as SetLeaves, have been
+deprecated and will be deleted in the near future. The semantic of WriteLeaves
```

Comment:
nit: in English saying "the semantics of X are" is more common than "the semantic of X is".